front/src/components/UI/Header/HeaderStyle.css

```diff
@@ -0,0 +1,58 @@
+.main-panel{
+  color:white;
+  width: 100%;
+  height: 60px;
+} 
+
+.panel-element{
+  width: 33.333%;
+  height: 100%;
+  background-color: rgb(0, 0, 0);
+  position: relative;
+  float:left;
+}
+
+.panel-center{
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.panel-image-left{
+  width: 30%;
+}
+
+.panel-image-right{
+  width: 30%;
+  filter: grayscale(100%);
+}
+
+@media (prefers-reduced-motion: no-preference) {
```

Comment:
Jeśli z tymi elementami będziesz miał problem żeby przenieść do styled componentów to daj znać, bo to może wykraczać lekko ponad podstawy i nie chcę cię wystraszyć